docs/install/harvester-configuration.md

```diff
@@ -91,6 +91,20 @@ For instance, if you have in your Harvester Configuration file specified `os.hos
 
 :::
 
+### `scheme_version`
+
+#### Definition
+
+The version of scheme is reserved for the future configuration migration.
+
+This configuration is mandatory for migrating configuration to new scheme version. It tells the Harvester the previous version and need to migrate.
```

Comment:
```suggestion
This configuration is mandatory for migrating the configuration to a new scheme version. It tells Harvester the previous version and the need to migrate.
```

Reply:
Please double-check if I understand it correctly ^

Reply:
It is correct.
thanks for correction.